.github/workflows/ci.yml

```
@@ -18,6 +18,7 @@ jobs:
       with:
         filters: |
           md: [ '**.md' ]
+          yml: [ '**.yml, '**.yaml' ]
```

I think that, you should add here also `.yamlint` file
```suggestion
          yml: [ '**.yml, '**.yaml', '.yamlint' ]
```